### ISSUE TITLE
CVE-2020-15945: impressive package information

### ISSUE THREAD

**Issue by xnox** [wolfi-dev/advisories#9206]
**Description**

https://images.chainguard.dev/security/CVE-2020-15945

Expand references, and navigate to debian tracker at https://security-tracker.debian.org/tracker/CVE-2020-15945

lua5.3 is not-affected, as the bug is specific to 5.4.0

Thus status should be package lua5.4 not affected, code not present as Wolfi has never shipped v5.4.0

**Comment by eslerm** [comment]
Agreed: https://github.com/lua/lua/commit/a2195644d89812e5b157ce7bac35543e06db05e3#commitcomment-150387752

**Comment by eslerm** [comment]
In a conversation with MITRE and the upstream author, Roberto confirmed that the affected range of CVE-2020-15945 is since 5.4.0 and until 5.4.1 and referenced this bug/range as: https://www.lua.org/bugs.html#5.4.0-8 

Hopefully the MITRE CNA will update their CVE entry.

Many thanks Roberto.
